lambda.py

```diff
@@ -30,15 +35,24 @@ def json2kinto(event, context):
                    (i.e: schemas.json)
     """
 
-    args = []
+    args = {}
+
+    # Upgrade by reading some ENV variables
+    for key, arg in JSON2KINTO_ENV_VARIABLES.items():
+        value = os.getenv(key)
+        if value is not None:
+            args[arg] = value
 
     for key, value in event.items():
         if key in JSON2KINTO_ARGS:
-            args.append('--' + key)
-            args.append(value)
-
-    print("importer args", args)
-    importer_main(args)
+            args['--%s' % key] = value
+
+    flatten_args = list(reduce(lambda x, y: x + y, args.items()))
+    # Remove password from there
+    print("importer args", list(reduce(lambda x, y: x + y,
+                                       [x for x in args.items()
+                                        if x[0] not in JSON2KINTO_ENV_VARIABLES.values()])))
```

Comment:
Oh, I see, you mean so that you can visually confirm that the parameters are correct.